### ISSUE TITLE
Visually indicate mission quadrants and regions on table

### ISSUE THREAD

**Issue by datanoh** [trekcc-ops/gemp-stccg#71]
Reported by @andrewd18. When a user chooses to seed a mission on the left end of the spaceline, it is actually placed one from the left.

I'm suspicious this is working as intended, with confusion introduced by missions in different regions or quadrants. (I've been confused by that myself.) If this does turn out to be working correctly, it'd still be useful to address this, as the UI seems confusing for players.

**Comment by andrewd18** [comment]
This issue has disappeared on the `seed-phase-stable` branch.

**Comment by datanoh** [comment]
Worth moving it to a lower-priority UI issue even if it's resolved. I think the current implementation is a bit clunky. Ideally quadrants and regions would be visually demarcated on the table, and it would be clearer to the user when they're adding a mission to an existing quadrant/region or starting a new one.

**Comment by andrewd18** [comment]
Confirming this is due to me accidentally putting [Access Relay Station](https://www.trekcc.org/1e/?cardID=519) in one of my test decks, which is in a different quadrant. I agree we should try to visually indicate quadrants somehow, even for MVP.

Renaming this issue.

**Comment by andrewd18** [comment]
First-in-quadrant and last-in-quadrant CSS classes exist and are applied correctly. However, there is no value in the applied CSS that distinguishes one quadrant from another, so there's no easy way to color code or label the quadrants. I could add thicker borders or something, but that seems less useful than doing the work to add a color or label text.

There is no game UI indicator for region, that will have to be added from scratch.

**Comment by datanoh** [comment]
@andrewd18 Short-term fix for both of these issues is to remove the cards that cause them. None of the cards in the TNG precon decks, the TNG expansion, or the TNG: Supplemental expansion require grouping by region or quadrant. There's only 4 cards in the current card defs driving this issue, and we can leave them all out for MVP.

**Comment by datanoh** [comment]
@andrewd18 I made an update to the stop_event branch to support separating regions in the client. For the game events that include a mission's quadrant as an attribute, they'll also have "region" (if the mission is in one). Those events will also have the "serializedGameState" attribute that includes quadrant and region for all locations in the game.

Added it to the stop_event branch; right now, it's the only branch passing the serializedGameState along. It'd probably be good for us to merge stop_event ASAP even if we just merge it as is. Any thoughts on that?
